.circleci/config.yml

```diff
@@ -125,6 +139,132 @@ jobs:
       - run-bazel-rbe:
           command: bazel test //test-end-to-end:test-end-to-end --test_output=streamed --spawn_strategy=standalone
 
+  test-assembly-mac-zip:
+    macos:
+      xcode: "9.0"
+    working_directory: ~/grakn
+    steps:
+      - checkout
+      - install-bazel-mac
+      - run:
+          command: |
+            bazel build //:assemble-mac-zip
+            unzip bazel-genfiles/grakn-core-all-mac.zip -d bazel-genfiles/
+            nohup bazel-genfiles/grakn-core-all-mac/grakn server start
+      - run: bazel test //test-deployment:test-deployment --test_output=streamed --spawn_strategy=standalone --cache_test_results=no
+      - run: bazel-genfiles/grakn-core-all-mac/grakn server stop
+
+# TODO: add after fixing the python issue regarding missing bdist_wheel
+#  test-assembly-windows-zip:
```

Comment:
Should we fix the python issue first so we can re-enable this? or is it too big and we want to skip it for now?

Reply:
Yes, I want to submit a follow-up PR with other improvements to the test as a whole